sktime/benchmarking/forecasting.py

```diff
@@ -28,14 +30,46 @@ def forecasting_validation(
         Each BaseMetric output will be included in the results.
     estimator : BaseForecaster object
         Estimator to benchmark.
-
+    backend : {"dask", "loky", "multiprocessing", "threading"}, by default None.
+        Runs parallel evaluate for each task if specified.
+
+        - "None": executes loop sequentally, simple list comprehension
+        - "loky", "multiprocessing" and "threading": uses ``joblib.Parallel`` loops
+        - "joblib": custom and 3rd party ``joblib`` backends, e.g., ``spark``
+        - "dask": uses ``dask``, requires ``dask`` package in environment
+        - "dask_lazy": same as "dask",
+        but changes the return to (lazy) ``dask.dataframe.DataFrame``.
```

Comment:
I think this needs to be indented to render correctly.

Also, will it run the benchmark lazily, or eagerly?

Reply:
I suppose that the docs are completely missing or? 

I think the benchmarks will be executed if `run` is called.

Reply:
oh, I see, there is indeed no docs page for this! We need to add to the API referenc, but that seems like a separate issue.

Reply:
#6649